arangod/RestHandler/RestCollectionHandler.cpp

```diff
@@ -437,21 +438,24 @@ void RestCollectionHandler::handleCommandPut() {
             res = trx->begin();
 
             if (res.ok()) {
-              auto result = trx->truncate(coll->name(), opts);
+              OperationResult result = trx->count(coll->name(), transaction::CountType::Normal);
```

Comment:
How expensive is a `count` compared to a truncate on large collections?

Reply:
_count_ should be instant, because it is just reading a cached value.